simulator/mips/mips_register/mips_register.def

```diff
@@ -63,4 +63,36 @@ CP0_REGISTER(CacheErr),
 CP0_REGISTER(TagLo),
 CP0_REGISTER(TagHi),
 CP0_REGISTER(ErrorEPC),
-CP0_REGISTER(DESAVE),
\ No newline at end of file
+CP0_REGISTER(DESAVE),
+CP1_REGISTER(f0),
+CP1_REGISTER(f1),
+CP1_REGISTER(f2),
+CP1_REGISTER(f3),
+CP1_REGISTER(f4),
+CP1_REGISTER(f5),
+CP1_REGISTER(f6),
+CP1_REGISTER(f7),
+CP1_REGISTER(f8),
+CP1_REGISTER(f9),
+CP1_REGISTER(f10),
+CP1_REGISTER(f11),
+CP1_REGISTER(f12),
+CP1_REGISTER(f13),
+CP1_REGISTER(f14),
+CP1_REGISTER(f15),
+CP1_REGISTER(f16),
+CP1_REGISTER(f17),
+CP1_REGISTER(f18),
+CP1_REGISTER(f19),
+CP1_REGISTER(f20),
+CP1_REGISTER(f21),
+CP1_REGISTER(f22),
+CP1_REGISTER(f23),
+CP1_REGISTER(f24),
+CP1_REGISTER(f25),
+CP1_REGISTER(f26),
+CP1_REGISTER(f27),
+CP1_REGISTER(f28),
+CP1_REGISTER(f29),
+CP1_REGISTER(f30),
+CP1_REGISTER(f31),
```

Comment:
Please add a new line here (old file editors are confused if there is no new line before EOF)